### PR TITLE
Update WhyNotWin11.au3

### PR DESCRIPTION
**Pull request by Qazy147**
Colors for results in one place (functions).

Line 606: multi-line string for translation
Line 717: no translatin, "DirectX 12 and WDDM 3" in LANG-file
Line 724: No translatin, "DirectX 12 and WDDM 2" in LANG-file

Change on line 463 is mistake made by codepage.

### CODE REVIEW

File: WhyNotWin11.au3 [hunk 1]
```diff
@@ -460,7 +460,7 @@ Func Main()
 	GUICtrlSetColor(-1, 0xE20012)
 	GUICtrlSetFont(-1, $aFonts[$FontMedium] * _GDIPlus_GraphicsGetDPIRatio()[0])
 
-	GUICtrlCreateLabel("❌", 760, 10, 30, 30, $SS_CENTER + $SS_CENTERIMAGE)
+	GUICtrlCreateLabel("âťŚ", 760, 10, 30, 30, $SS_CENTER + $SS_CENTERIMAGE)
```
Comment (rcmaehl) [inline WhyNotWin11.au3:463]: I'm unsure why this is needed? Is this not displaying for some languages?

File: WhyNotWin11.au3 [hunk 2]
```diff
@@ -1052,4 +1005,19 @@ EndFunc   ;==>_SetFile
 Func _Translate($iMUI, $sString)
 	_INIUnicode(@LocalAppDataDir & "\WhyNotWin11\Langs\" & $iMUI & ".lang")
 	Return IniRead(@LocalAppDataDir & "\WhyNotWin11\Langs\" & $iMUI & ".lang", "Strings", $sString, $sString)
-EndFunc   ;==>_Translate
\ No newline at end of file
+EndFunc   ;==>_Translate
+
+Func _GUICtrlSetYes($id)
```
Comment (rcmaehl) [inline WhyNotWin11.au3:1010]: Please change to _GUICtrlSetPass

File: WhyNotWin11.au3 [hunk 3]
```diff
@@ -1052,4 +1005,19 @@ EndFunc   ;==>_SetFile
...
+	GUICtrlSetData($id, "OK")
+	GUICtrlSetBkColor($id, 0x4CC355)
+EndFunc   ;==>_GUICtrlSetYes
+
+Func _GUICtrlSetNo($id)
```
Comment (rcmaehl) [inline WhyNotWin11.au3:1015]: Please change to _GUICtrlSetFail

File: WhyNotWin11.au3 [hunk 4]
```diff
@@ -1052,4 +1005,19 @@ EndFunc   ;==>_SetFile
...
+	GUICtrlSetData($id, "X")
+	GUICtrlSetBkColor($id, 0xFA113D)
+EndFunc   ;==>_GUICtrlSetNo
+
+Func _GUICtrlSetMaybe($id, $symbol = "?")
```
Comment (rcmaehl) [inline WhyNotWin11.au3:1020]: Please change to _GUICtrlSetWarn

**Review by rcmaehl — CHANGES_REQUESTED** [review]
Change to Request Changes

File: WhyNotWin11.au3 (hunk 1 above)
Comment (Qazy147) [reply WhyNotWin11.au3:463]: It's not needed it is my mistake (wrong codepage). I would  replace it by `ChrW(0x274C)`.

**Pull request closed without merging**